### ISSUE TITLE
off-site links should open in new tabs

### ISSUE THREAD

**Issue by jchris** [web3-storage/web3.storage#2006]
eg the link to "Status" and "Open an issue" on the account page, should open in a new tab

- [ ] if possible, make all off-site links open in new tab by default.
- [x] ensure the "status" link in footer opens in new tab
- [x] ensure the "open an issue" link in footer opens in new tab

**Comment by gobengo** [comment]
I'll make a change for the footer links via html. The 'make all off-site links open in new tab by default' could be done in js but idk it may come at the cost of overriding the default user-agent behavior around opening links so I'm more hesitant (e.g. could create accessibility issues).

**Comment by gobengo** [comment]
@jchris
>  if possible, make all off-site links open in new tab by default.

@alanshaw and I don't wanna do this unless you insist. So we're just going to do this for footer links for now.

**Comment by gobengo** [comment]
@jchris I tested on prod the two links in footer you mentioned, and I believe they work as desired. 

We did not do `if possible, make all off-site links open in new tab by default.`, explained in previous comment. But we now have better tools in the codebase to add this 'new tab for external links' to various parts of the site in a controlled fashion. Feel free to make followup issue for any other links not opening how you want.
